client/src/components/NavBar/NotificationsMenu/NotificationsMenu.tsx

```diff
@@ -0,0 +1,113 @@
+import useStyles from './useStyles';
+import { useAuth } from '../../../context/useAuthContext';
+import {
+  Typography,
+  Button,
+  Badge,
+  MenuItem,
+  Menu,
+  Paper,
+  ListItemText,
+  ListItemAvatar,
+  Avatar,
+  ListItem,
+  Box,
+} from '@material-ui/core';
+import { Notification } from '../../../interface/Notification';
+import React, { useEffect, useState } from 'react';
+import { getAllUnreadNotifications, markAsRead } from '../../../helpers/APICalls/Notification';
+
+export default function NotificationsMenu(): JSX.Element {
+  const classes = useStyles();
+  const { loggedInUser } = useAuth();
+
+  const [anchorEl, setAnchorEl] = React.useState<null | HTMLElement>(null);
+
+  const handleClick = (event: React.MouseEvent<HTMLButtonElement>) => {
+    notifications?.map(async (notification) => {
+      await markAsRead(notification._id);
+    });
+
+    setAnchorEl(event.currentTarget);
+  };
+
+  const handleClose = () => {
+    setAnchorEl(null);
+  };
+  const [notifications, setNotifications] = useState<[Notification]>();
+  useEffect(() => {
+    async function fetchNotifications() {
+      if (loggedInUser) {
+        const fetchedNotifications = await getAllUnreadNotifications();
+        setNotifications(fetchedNotifications);
+      }
+    }
+    fetchNotifications();
+  }, [loggedInUser]);
+
+  return (
+    <div className={classes.menuButton}>
+      <Button aria-controls="simple-menu" aria-haspopup="true" onClick={handleClick}>
+        <Badge variant="dot" color="primary" invisible={notifications && notifications.length < 1}>
+          <Typography variant="button">Notifications</Typography>
+        </Badge>
+      </Button>
+      {notifications && notifications.length >= 1 && (
+        <Menu
+          className={classes.menu}
+          elevation={0}
+          getContentAnchorEl={null}
```

Comment:
Is this prop necessary/does it contribute?

Reply:
I kept them because whenever I tried to change or remove `getContentAnchorEl` or  `elevation`. the drop-down menu would cover up the navigation buttons